notifier/metric.go

```diff
@@ -52,18 +53,26 @@ func newAlertMetrics(r prometheus.Registerer, queueCap int, queueLen, alertmanag
 		},
 			[]string{alertmanagerLabel},
 		),
-		dropped: prometheus.NewCounter(prometheus.CounterOpts{
+		retries: prometheus.NewCounterVec(prometheus.CounterOpts{
+			Namespace: namespace,
+			Subsystem: subsystem,
+			Name:      "retries_total",
+			Help:      "Total number of send retries after failure.",
+		},
+			[]string{alertmanagerLabel},
+		),
```

Comment:
I think this is a leftover from a previous implementation when I was implementing a retry functionality.
I'll drop it since we don't want to change Prometheus' behavior. When sending to an Alertmanager fails, alerts buffered for that Alertmanager will be dropped.